### DISCUSSION TITLE
Interested in contributing but not exactly sure about the concept of the library

### DISCUSSION THREAD

**Discussion by pilcrowonpaper** [svelteuidev/svelteui#30] [General]
Hi,

I’m interested in contributing, but I’m not sure what makes this different from the number of similar libraries out there. Or rather, since it’s still in beta, how it will be different. Personally, I’m interested in a minimalistic minimal UI library that you can customize (with actions) for prototyping/demos. But, most libraries out there look too generic and you need css for creating simple layouts. If this library can address those problems, I would love to contribute. 

-thanks

**Comment by kamellperry** [comment]
This library not only aims to be more than just a component library, but one that fits all the needs of a project. On the component side though, no css is required to have the base styling. You can install the package and start using the components with zero configuration. I think this library in the Svelte community specifically will be different.
